.circleci/integration-tests/master_dag.py

```diff
@@ -70,6 +70,7 @@ def get_report(dag_run_ids: List[str], **context: Any) -> None:
             f"*{header}:* `{value}`\n"
             for header, value in [
                 ("Runtime version", os.getenv("ASTRONOMER_RUNTIME_VERSION", "N/A")),
+                ("Python version", os.getenv("PYTHON_VERSION", "N/A")),
```

Comment:
I'm good with this change, but may I know where is this `PYTHON_VERSION` set?

Reply:
@Lee-W its set in runtime image
<img width="640" alt="image" src="https://github.com/astronomer/astronomer-providers/assets/43964496/2c269421-7edb-45eb-830d-b1573d0d1cbc">